packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -132,6 +132,64 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
   };
 
   const seoComponent = <NextSeo {...seo} />;
+  const schemaComponent = (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
+        __html: JSON.stringify({
+          '@context': 'https://schema.org',
+          '@type': 'Article',
+          headline: post?.title,
+          dateCreated: post?.createdAt,
+          datePublished: post?.createdAt,
+          dateModified: post?.updatedAt,
```

Comment:
This code looks ugly. But in sake of being scrappy, I'd rather merge what we have now. We can always improve the code quality in a followup PR.